From: Ben Hutchings <bhutchings@solarflare.com>
Date: Fri, 27 Jul 2012 20:50:57 +0100
Subject: sfc: Remove bogus comment about MTU change and RX buffer overrun
Patch-mainline: v3.7-rc1
Git-commit: 3f65ea5b2aa0f76a463c85ed39955c26faf4f5a7
References: bnc#786035 FATE#314299

RX DMA is limited by the length specified in each descriptor and not
by the MAC.  Over-length frames may get into the RX FIFO regardless of
the MAC settings, due to a hardware bug, but they will be truncated by
the packet DMA engine and reported as such in the completion event.

Signed-off-by: Ben Hutchings <bhutchings@solarflare.com>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/sfc/efx.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ethernet/sfc/efx.c
+++ b/drivers/net/ethernet/sfc/efx.c
@@ -1959,8 +1959,6 @@ static int efx_change_mtu(struct net_dev
 	efx_stop_all(efx);
 
 	mutex_lock(&efx->mac_lock);
-	/* Reconfigure the MAC before enabling the dma queues so that
-	 * the RX buffers don't overflow */
 	net_dev->mtu = new_mtu;
 	efx->type->reconfigure_mac(efx);
 	mutex_unlock(&efx->mac_lock);
